server/functions/formatting.go

```diff
@@ -0,0 +1,1411 @@
+// Copyright 2025 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package functions
+
+import (
+	"math"
+	"strconv"
+	"strings"
+	"time"
+	"unicode"
+
+	"github.com/dolthub/doltgresql/postgres/parser/duration"
+
+	"github.com/cockroachdb/errors"
+	"github.com/dolthub/go-mysql-server/sql"
+	errors2 "gopkg.in/src-d/go-errors.v1"
```

Comment:
I think we've mostly moved to `"github.com/cockroachdb/errors"` for Doltgres errors, rather than the GMS error package.